src/frontend/src/test-e2e/views.ts

```diff
@@ -7,6 +9,7 @@ export class WelcomeView extends View {
     await this.browser
       .$("#registerUserNumber")
       .waitForDisplayed({ timeout: 10_000 });
+    await waitForImages(this.browser);
```

Comment:
extra nit: I guess this will always be useful for screenshots. Should we add this to the [screenshot taking function](https://github.com/dfinity/internet-identity/blob/59c06f609955eabb136e2968be7070bedd511ddb/src/frontend/src/test-e2e/util.ts#L328)?

Reply:
👍